doc/user/interop.md

```diff
@@ -200,6 +200,25 @@ Otherwise throws `UnknownIdentifierException`.
 
 In all cases where a call is made no block is passed.
 
+
+### `REMOVE`
+
+The name must be an `int` (small Ruby `Fixnum`), or a Ruby `String` or `Symbol`,
+or a Java `String`.
```

Comment:
I was going to tell you that this is wrong - but now I see that it's what I've written elsewhere.

It's wrong because by the time Ruby responds to the message (which is what this section is about) it's already just a Java `String` if it used to be a Ruby `String` or `Symbol`.

Leave it in and I'll update afterwards.